types/artifacts.d.ts

```diff
@@ -240,6 +240,8 @@ declare global {
         crossOrigin: string | null
         /** Where the link was found, either in the DOM or in the headers of the main document */
         source: 'head'|'body'|'headers'
+        /** Path that uniquely identifies the node in the DOM */
+        devtoolsNodePath?: string
```

Comment:
should this always be defined since we only look at link elements in the document?

Reply:
Hey I was thinking because `LinkElements` can also be collected from network resource headers, I guess we don't want a devtools path there, wdyt? https://github.com/GoogleChrome/lighthouse/blob/e040b125326e258741214880ad7ed3e899fe9c3f/lighthouse-core/gather/gatherers/link-elements.js#L104

Reply:
> Hey I was thinking because `LinkElements` can also be collected from network resource headers, I guess we don't want a devtools path there, wdyt?

whoops, yes, you're absolutely right